google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -169,7 +169,9 @@ appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
 appengine.deployment.deployallconfigs.checkbox.label=Update dos, dispatch, cron, queues, and datastore indexes
 appengine.deployment.flexible.nogaesupport=No modules with Google App Engine flexible framework support found.
 appengine.deployment.error.cancelled=Deployment process was cancelled.
+appengine.deployment.staging.artifact.as=Staging artifact as: {0}
 appengine.converter.description=Convert projects created by the legacy App Engine plugin to be compatible with the new plugin
+appengine.staged.artifact.name.converter.description=Converts run configurations to set a legacy bit for preserving the old staged artifact name.
```

Comment:
Is this message visible to users? If so maybe update the wording - its clear to us but may be too cryptic for users.

Reply:
I just realized that this is visible if the user selects the `Details...` link in the conversion dialog. What do you suggest?

Reply:
Something like this maybe?

`Converter for deployment run configs that marks the deployment as using target.jar/war for the staged artifact name in App Engine flexible deployments`

Its not very elegant, but something along those lines.

Reply:
Considering this statement is prefixed with `The following conversions will be performed:`, what about:

> App Engine flexible deployment run configurations will have their staged artifact name explicitly set to target.jar/war

Reply:
I'd add "existing" to the front of the sentence.

Also consider the following added to the end (feel free to ignore if you think its too verbose):

`Existing App Engine flexible deployment run configurations will have their staged artifact name explicitly set to target.jar/war for interoperability with existing Dockerfiles.`

Reply:
Done.